### DISCUSSION TITLE
上传到多个图床

### DISCUSSION THREAD

**Discussion by Starrynightzyq** [Molunerfinn/PicGo#706] [Ideas]
PicGo能不能支持同时上传到多个图床？比如一张图片，同时上传到腾讯云COS、GitHub和本地的NAS，这样互为备份，总觉得把图片传到别人的服务器上不安全，希望有一个本地备份。

**Comment by Molunerfinn** [comment]
目前的架构暂时不支持一次性上传多个图床。未来会考虑支持

**Comment by sosel** [comment]
好期待，就算某个图床挂了，能直接在数据库中替换自定议域名，满血复活！

**Reply by Molunerfinn** [reply]
目前批量替换 URL 已经支持
